src/app/app.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  run = 0
+
+  onStart(ev){
+    this.run = ev
+  }
+}

```

Comment:
what is do 'ev', 'run', 'onStart' mean? replace all that with self-declarative names